webpack.config.js

```diff
@@ -15,7 +15,8 @@ module.exports = {
     project: './src/main/webapp/project/index.js',
     projectDocument: './src/main/webapp/projectDocument/index.js',
     projectReview: './src/main/webapp/projectReview/index.js',
-    search: './src/main/webapp/search/index.js'
+    search: './src/main/webapp/search/index.js',
+    linkWizard: './src/main/webapp/linkWizard/index.js'
```

Comment:
This should be alphabetized for clarity. We've had cases where duplicates were defined, but out of order, so it was harder to see.